content/php/concepts/string-functions/terms/htmlentities/htmlentities.md

```diff
@@ -0,0 +1,121 @@
+---
+Title: 'htmlentities()'
+Description: 'Converts all applicable characters to HTML entities.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`htmlentities()`** function converts characters to HTML entities. To convert HTML entities back to characters, use the `html_entity_decode()` function.
+
+
+## Syntax
+
+```pseudo
+htmlentities(string, flags, encoding, double_encode)
+```
+
+**The predefined characters are**:
+
+|Characters|Convert to|Result|
+|---|---|---|
+|& (ampersand)|becomes|\&amp;|
+|" (double quote)|becomes|\&quot;|
+|' (single quote)|becomes|\&#039;|
+|< (less than)|becomes|\&lt;|
+|> (greater than)|becomes|\&gt;|
+
+
+The `htmlentities()` function has one required parameter and three optional parameters:
+
+- `$string`: Required. Specifies the `string` to convert.
```

Comment:
I think we should move the parameter descriptions together, and move all the constant tables after the syntax section into their own section.